website/docs/r/is_virtual_endpoint_gateway.html.markdown

```diff
@@ -71,9 +81,9 @@ Review the argument references that you can specify for your resource.
 - `target` - (Required, List) The endpoint gateway target.
 
   Nested scheme for `target`:
-  - `crn` - (Optional, Forces new resource, String) The endpoint gateway target `CRN`. If CRN not specified, `name` must be specified. 
+  - `crn` - (Optional, Forces new resource, String) The CRN for this provider cloud service, or the CRN for the user's instance of a provider cloud service.. If CRN not specified, `name` must be specified. 
   - `name` - (Required, Forces new resource, String) The endpoint gateway target name.
-  - `resource_type` - (Required, String) The endpoint gateway target resource type.
+  - `resource_type` - (Required, String) The endpoint gateway target resource type, allowed values [ provider_cloud_service, provider_infrastructure_service ].
```

Comment:
Can we use the following style to to show the enums and remove [ ] ?

The possible values are `cpu`, `memory`, `network_in` and `network_out`.

Reply:
Incorporated this comment